app/components/account-settings/security/mfa-switch-to-app/index.ts

```
@@ -0,0 +1,182 @@
+  }
+
+  staInitialEmail = task(async () => {
+    try {
+      await this.ajax.post(this.mfaEndpoint, {
+        data: {
+          method: ENUMS.MFA_METHOD.TOTP,
+        },
+      });
+    } catch (error) {
+      const errorObj = (error as AjaxError).payload || {};
+      const otpMsg = errorObj.otp && errorObj.otp[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)